docs/pages/networking.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Networking
```

Comment:
This should be more like `Usage` or `Operation` and should be merged with `run-time-settings.md`.
The fact that we interact with it over the network is a detail.